src/app/modules/shared/components/dictionaries/manufacturer/manufacturer.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-card>
+
+  <div>
+    <h2 class="text-center py-3 m-0">Виробники</h2>
+  </div>
+
+  <div>
+    <app-create-manufacturer *ngIf="isAdmin" (createManufacturer)="addManufacturer($event)"></app-create-manufacturer>
+  </div>
+
+  <ng-template #actionsTemplate let-data="templateData">
+
+    <app-delete-manufacturer *ngIf="isAdmin" [manufacturer]="data" (deleteManufacturer)="deleteManufacturer($event)">
```

Comment:
You have to set template for mat-fsp-table if needed, instead of some checks in template. In yours case analyst will see empty column for actions